recipes/chunkio/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class ChunkIOConan(ConanFile):
+    name = "chunkio"
+    description = "Simple library to manage chunks of data in memory and file system"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/edsiper/chunkio"
+    topics = ("chunk", "io", "memory", "file")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_filesystem": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_filesystem": True,
```

Comment:
This option is not being used in the recipe. Is it aimed at managing the `CIO_BACKEND_FILESYSTEM` cmake variable?